Buildings/Fluid/CHPs/Data/Generic.mo

```diff
@@ -0,0 +1,104 @@
+within Buildings.Fluid.CHPs.Data;
+record Generic "Generic data for CHP models"
+  extends Modelica.Icons.Record;
+  parameter Real[27] coeEtaQ=fill(0,27)
```

Comment:
If no default value can be provided, don't provide simply 0. Please remove all 0 that do not make sense.